tests/test_config.py

```diff
@@ -260,9 +284,10 @@ def test_sc_v2(self):
         specified.
 
         """
-        targets = (
-            "nailgun.config.ServerConfig(url='flim', auth='flam')",
-            "nailgun.config.ServerConfig(auth='flam', url='flim')",
+        targets = list(
+            "nailgun.config.ServerConfig({0})".format(', '.join(permutation))
+            for permutation
+            in permutations(("url='flim'", "auth='flam'", "version='1!0'"))
```

Comment:
There's no guarantee about which order `repr` might place arguments. Any of these are possible:

    "nailgun.config.ServerConfig(url='flim', auth='flam', version='1!0')"
    "nailgun.config.ServerConfig(url='flim', version='1!0', auth='flam')"
    "nailgun.config.ServerConfig(auth='flam', url='flim', version='1!0')"
    "nailgun.config.ServerConfig(auth='flam', version='1!0', url='flim')"
    "nailgun.config.ServerConfig(version='1!0', url='flim', auth='flam')"
    "nailgun.config.ServerConfig(version='1!0', auth='flam', url='flim')"